packages/odyssey-storybook/.storybook/preview-head.html

```diff
@@ -11,3 +11,11 @@
   href="https://fonts.googleapis.com/css2?family=Inconsolata&display=swap"
   rel="stylesheet"
 />
+
+<style>
+  .test-class.test-class {
+    background-color: rebeccapurple;
+    color: white;
+    padding: 10px;
+  }
+</style>
```

Comment:
this is just testing code that will be removed? Or do you intend to merge this?

Reply:
I'd like to be able to show an example of a custom class being applied to box, if I import a css style in the story would that be a better way?

Reply:
a story decorator might be easier to track/maintain

Reply:
I see, hmmmm yea this works then - doesn't require building any CSS which I like